internal/orchestrion/context_stack.go

```diff
@@ -17,14 +17,14 @@ func getDDContextStack() *contextStack {
 		return gls.(*contextStack)
 	}
 
-	newStack := new(contextStack)
+	newStack := &contextStack{}
```

Comment:
You need to `make(contextStack)` to actually initialize the map. So maybe the complete form is 

```suggestion
	newStack := &contextStack{}
	*newStack = make(contextStack)
```

Reply:
:thinking: I tested it it should be working

Reply:
D'oh, my mistake! It does indeed work. Sorry for the noise. I didn't realize `&contextStack{}` actually initialized the map. TIL!